packaging/debian/build.sh

```
@@ -18,25 +19,40 @@ echo "Installing required dependencies..."
-# Clean previous build
-rm -rf "$BUILD_DIR" *.deb
+# Get project root directory (parent of packaging directory)
+PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+
+# Normalize OUTPUT_DIR to absolute path
+if [[ "$OUTPUT_DIR" != /* ]]; then
+    OUTPUT_DIR="$PROJECT_ROOT/$OUTPUT_DIR"
+fi
+OUTPUT_DIR="$(mkdir -p "$OUTPUT_DIR" && cd "$OUTPUT_DIR" && pwd)"
+
+rm -rf "$PROJECT_ROOT/$BUILD_DIR"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)